logs_sdk/lib/opentelemetry/sdk/logs/export.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Logs
+      # The export module contains result codes for LoggerProvider#force_flush
```

Comment:
These _should_ also be result codes for exporters - their use shouldn't just relate to these two methods.

Reply:
Absolutely. I was being overly specific in the class description with the intent of making it more general when there were more places the constants were used. 

Updated in: 5528277401c8354cfab8ade8a17f4d65b20f7dd9

Reply:
Great point. Updated to be more general in: d63a4e9dc1ce01b6d63914dc7e68a235027048d3